packages/react-dom/src/events/__tests__/SyntheticClipboardEvent-test.js

```diff
@@ -9,24 +9,39 @@
 
 'use strict';
 
-var SyntheticClipboardEvent;
+var jsdom;
+var React;
+var ReactDOM;
 
 describe('SyntheticClipboardEvent', () => {
-  var createEvent;
+  var document;
+  var simulateEvent;
 
   beforeEach(() => {
-    // TODO: can we express this test with only public API?
-    SyntheticClipboardEvent = require('../SyntheticClipboardEvent');
-    createEvent = function(nativeEvent) {
-      var target = require('../getEventTarget')(nativeEvent);
-      return SyntheticClipboardEvent.getPooled({}, '', nativeEvent, target);
+    jsdom = require('jsdom').jsdom;
+    React = require('react');
+    ReactDOM = require('react-dom');
+
+    document = jsdom('');
```

Comment:
Why do we need this? Don't we already have a `document` global?

Reply:
To reset changes done on `document.body`, replaced it by `document.body.innerHTML = ''`.